ROADMAP.md

```diff
@@ -0,0 +1,40 @@
+Roadmap
+=======
+
+These are the themes that we plan to work on for cert-manager. If you wish
+to discuss these topics you can find us in #cert-manager on Kubernetes Slack, or
+at our [community meetings](https://cert-manager.io/docs/contributing/#meetings).
+
+The roadmap items are categorized in to themes based on the larger goals we
+want to achieve with cert-manager.
+
+* Beyond Ingress: improve experience of cert-manager for applications beyond just
+  ingress certificates
+  * Service Mesh Integration: Enable service meshes to issue mTLS certificates
+    with cert-manager, getting the integration with external issuers and the
+    audit capabilities of cert-manager in their mesh
+  * Istio agent certificates issued via cert-manager
+  * CSI driver: seamlessly deliver unique certs + keys to workloads
+* Adoption of upstream APIs: continue to support latest APIs for k8s upstream
+  * CSR API: support CSR API as a standard for certificate requests in kubernetes
+* Policy: giving cluster admin granular control over certificate issuance
+  * Extensible primitives within cert-manager for defining policy for
+    acceptable CertificateRequests
+* Extensibility: widen the scope of integrations with cert-manager
+  * EST support: support a standard for ACME-like issuance within an enterprise
+  * External DNS plugin: enable ACME DNS01 requests to be completed using external-dns
+  * OpenShift Routes support: provide similar capabilities to Ingress for
+    issuing certs
+  * Improve external issuer development experience: documentation and examples
```

Comment:
Add policy to this list

Reply:
The policy section already says that it should be extensible, so is it worth saying that you should be able to extend policy in the extensibility section?

Reply:
maybe not.. this was a remark from the meeting but maybe having it once is enough